packages/core/src/app/payment/PaymentForm.tsx

```diff
@@ -152,6 +164,7 @@ const PaymentForm: FunctionComponent<
                 ) : (
                     <PaymentSubmitButton
                         brandName={brandName}
+                        isCheckoutPaymentMethod={isCheckoutPaymentMethod}
```

Comment:
It might be better to update this code with:

```suggestion
                        isCheckoutPaymentMethod={!!selectedMethod?.initializationData?.isCheckoutPaymentMethod}
```